sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -40,18 +40,30 @@ Licensed under the MIT License.
     <!-- Configures the Java 9+ run to perform the required module exports, opens, and reads that are necessary for testing but shouldn't be part of the module-info. -->
     <javaModulesSurefireArgLine>
       --add-opens com.azure.cosmos/com.azure.cosmos=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.models=ALL-UNNAMED
```

Comment:
@kushagraThapar @FabianMeiswinkel these are additional opens needed now that testing uses module paths instead of class paths